.github/workflows/ci.yml

```diff
@@ -9,6 +9,22 @@ on:
       - '*'
 
 jobs:
+  gofmt:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2.1.4
+      with:
+        go-version: 1.16
+      id: go
+    # Check the codebase has been formatted by `gofmt` by checking whether
+    # the output of `gofmt -l` is empty, as recommended in
+    # https://github.com/golang/go/issues/24230)
+    - name: Run gofmt
+      run: test -z $(gofmt -l .)
```

Comment:
Can you add another `gofmt -l` call here so that the output (unformatted files) makes it into the CI log?
Currently, this only shows *that* some files are not formatted, not *which* files are not formatted.

Reply:
@stapelberg Sure. That's a good point. I actually added `gofmt -d` which prints diffs in addition to formatted files.